src/App.tsx

```diff
@@ -85,12 +79,22 @@ export const App: React.FC = () => {
             <div className="block">
               {!todos.length
                 ? <Loader />
-                : <TodoList todos={visibleTodos} viewModule={viewModule} />}
+                : (
+                  <TodoList
+                    todos={visibleTodos}
+                    viewModule={setSelectTodoId}
+                  />
+                )}
             </div>
           </div>
         </div>
       </div>
-      {selectTodoId && (<TodoModal selectTodo={selectTodo} viewModule={viewModule} />)}
+      {!!selectTodoId && (
```

Comment:
SelectTodoId = 0,  And without `"!!"` it rendered on the page as text

Reply:
Right, that's why I added my comment )
<img width="800" alt="Screenshot 2022-10-05 at 22 20 52" src="https://user-images.githubusercontent.com/47606379/194144478-70545fd0-4d84-479e-a412-f9063b7dbf77.png">